src/sections/Pricing/feature_data.json

```diff
@@ -445,7 +445,7 @@
       "teamOperator": "x",
       "enterprise": "x"
     },
-    "docs": ""
+    "docs": "https://docs.layer5.io/kanvas/advanced/performance/#optimize-your-design-using-the-layers-panel"
```

Comment:
This isn't the most appropriate reference. Seek out docs pages that describe features (not performance tuning). Do we have any? If not, this might be a gap that needs filled.

Thanks for working on this, @kishore08-07.